test/quantization/pt2e/test_graph_utils.py

```diff
@@ -94,3 +98,39 @@ def forward(self, x):
             m, [torch.nn.BatchNorm2d, torch.nn.ReLU]
         )
         self.assertEqual(len(fused_partitions), 0)
+
+    @unittest.skipIf(IS_WINDOWS, "torch.compile is not supported on Windows")
+    def test_customized_equivalet_types_dict(self):
+        native_equivalent_types = get_equivalent_types()
+        customized_equivalent_types = native_equivalent_types
+
+        for equivalent_type_set in customized_equivalent_types:
+            if torch.nn.ReLU in equivalent_type_set:
+                equivalent_type_set.clear()
+                equivalent_type_set.add(torch.nn.ReLU)
```

Comment:
Not sure what this test is doing really. You should check for some custom type.

Reply:
Sure, changed.